package.json

```diff
@@ -63,6 +63,8 @@
 		"postrn-bundle": "patch-package --reverse --patch-dir gutenberg/packages/react-native-editor/metro-patch",
 		"i18n:gutenberg": "./bin/i18n-download.sh gutenberg",
 		"i18n:gutenberg:force": "cross-env REFRESH_I18N=1 ./bin/i18n-download.sh gutenberg",
+		"i18n:update": "./bin/i18n-update.sh jetpack ./jetpack/projects/plugins/jetpack/extensions layout-grid ./block-experiments/blocks/layout-grid",
```

Comment:
Eventually, once this script is ready to merge, we could remove all the `makepot` and `wpcli` commands and attach this one to `prebundle:js`.